README.md

```diff
@@ -6,13 +6,11 @@
 [![Discord Channel](https://img.shields.io/discord/489522408076738561.svg)](https://discord.gg/AJSX9DCSUA)
 [![Open Source Helpers](https://www.codetriage.com/ocelot-social-community/ocelot-social/badges/users.svg)](https://www.codetriage.com/ocelot-social-community/ocelot-social)
 
-<!-- [ -->Ocelot.social<!-- ](<https://ocelot.social>) --> is free and open source software program code to run social networks. Its development is supported by a community of programmers and interested network operators.
+[Ocelot.social](https://ocelot.social) is free and open source software program code to run social networks. Its development is supported by a community of programmers and interested network operators.
 
 <!-- markdownlint-disable MD033 -->
 <p align="center">
-  <!-- <a href="https://ocelot.social" target="_blank"> -->
-    <img src="https://raw.githubusercontent.com/Ocelot-Social-Community/Ocelot-Social/master/webapp/static/img/custom/logo-squared.svg" alt="ocelot.social" width="40%" height="40%">
-  <!-- </a> -->
+  <a href="https://ocelot.social" target="_blank"><img src="webapp/static/img/custom/logo-squared.svg" alt="ocelot.social" width="40%" height="40%"></a>
```

Comment:
The embedded images (from relative links) cuase the problem.
I'll take care of that.

Reply:
Good finding. 👍🏼🙏🏼 @mahula 

I was unable to derive that from the error massage, I have to admit.

Reply:
Fixed.